### PR TITLE
isolate direction of username within modlog

### PR DESCRIPTION
**Pull request by Jan200101**
Came up on the Discord as something that could possibly be implemented

Before this PR:
image

After this PR:
image

### CODE REVIEW

**Comment by Jan200101** [comment]
Force pushed first time to correct the styling checks

Force pushed the second time after I was asked if the second LEFT-TO-RIGHT was really needed, which I had some hours ago concluded it was but came to realize it really isn't

**Review by mathstrains21 — DISMISSED** [review]

File: redbot/core/modlog.py [hunk 1]
```diff
@@ -466,8 +466,13 @@ async def message_content(self, embed: bool = True):
             else:
                 user = f"{self.last_known_username} ({self.user})"
```
Comment (Jackenmen) [inline redbot/core/modlog.py]: This needs to be updated as well.

**Review by Jackenmen — CHANGES_REQUESTED** [review]
This doesn't work in embeds:
image
image

From my testing, putting LTR override at the beginning of the text instead works just fine so I think this should be a good solution:
image

It *could* make sense to also add this LTR override for the moderator though, it is way less likely to be an issue for a moderator's name.

**Comment by Jackenmen** [comment]
Sorry, I did not realize my screenshot contained an example with two LTR overrides rather than just one at the beginning (which works fine as well). I just removed the second LTR override myself in the commit I just pushed.

**Comment by Jan200101** [comment]
image
Talked about this on Discord

Isolating the name should keep its direction intact while allowing the format of `name#discrim (id)` to stay intact

**Review by Jackenmen — APPROVED** [review]
Works great now, thanks for the PR!

cat-xmas

**Pull request merged**